### PR TITLE
 IOTile Variable Constants

### LINKED ISSUE #90: Add class to store IOTile Variable Constants
There is a number of stream variables that are hardcoded on any application interacting with IOTile Cloud. For example, 
  - Gateway Scan Begin: `5a00`
  - Gateway Scan End: `5a01`
  - Gateway Scan Failure: `5a03`
  - End of Trip Summary: `5a07`
  etc.

Let's add a proper set of constants for clients to avoid hard coding these stream variable IDs

### PR DESCRIPTION
**Pull request by lekosfmi**
closes #90

### CODE REVIEW

**Comment by dkarchmer** [comment]
5800 - Battery

File: src/models/iotile-sys-var.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  public temperature: string = '5023';
+
+  public battery: string = '5800';
+
+  public gatewayScanBegin: string = '5a00';
+  public gatewayScanEnd: string = '5a01';
+  public gatewayScanFailure: string = '5a03';
+  public endOfTripSummary: string = '5a07';
+
+  public rebootStream: string = '5c00';
```
Comment (dkarchmer) [inline src/models/iotile-sys-var.ts]: Rename to **reboot** (either rename everything to somethingStream, or don't use stream anywhere - as they are ALL streams)

File: src/models/iotile-sys-var.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export class IOTileSysVar {
+  public event: string = '5020';
```
[the remark this answers is not in the document]
Comment (dkarchmer) [reply src/models/iotile-sys-var.ts]: Also, please note that what we call an "Event" is a type of data, so we can have lots and lots of different streams represented as Events. You can therefore not call 5020 an "Event". As I described, `5a07` is also an Event stream.
So, if we wanted to keep `5020` here (and we don't), you will need to call it something like `accelerometerData` or `shockVibration` or something that explains what kind of event is this

File: src/models/iotile-sys-var.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export class IOTileSysVar {
```
[the remark this answers is not in the document]
Comment (lekosfmi) [reply src/models/iotile-sys-var.ts]: Got it. So System Variables are at the device level while User Variables are defined in the SG. Therefore, devices can have the same System Variables but may differs in User Variables depending on the device's SG.

**Pull request merged**